src/App.tsx

```diff
@@ -1,14 +1,51 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [edit, setEdit] = useState(false);
```

Comment:
```suggestion
  const [isEditing, setIsEditing] = useState(false);
```

Reply:
Read this article about bool variables naming https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
But the main issue here is that when we double-click on one todo, all of them became editable
![Uploading image.png…]()